taxi/forms.py

```diff
@@ -0,0 +1,52 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm, UserChangeForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class LicenceValidateMixin:
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]  # noqa
+
+        if not (
+            len(license_number) == 8
+            and license_number[0:3].isupper()
+            and license_number[0:3].isalpha()
+            and license_number[3:].isnumeric()
+        ):
+            raise ValidationError(
+                "The driver's license must be in the format 'XXX00000' "
+                "(3 uppercase letters and 5 numbers)"
+            )
+
+        return license_number
+
+
+class DriverLicenseUpdateForm(LicenceValidateMixin, forms.ModelForm):
+    class Meta:
+        model = Driver

```

Comment:
read about get_user_model() and it's benefits in such cases. It is not a mistake, I guess. Again - up to you

Reply:
These forms are for the 'Driver' class. Decided that at change of the user it will be necessary to do separate forms.
As you can see further, I use the user form for its intended purpose:
![image](https://user-images.githubusercontent.com/72568844/200883475-bda5ce3d-bffb-405a-b707-aef399a428a7.png)